.ameba.yml

```diff
@@ -13,3 +13,9 @@ Lint/NotNil:
   - src/selenium/driver.cr
   Enabled: true
   Severity: Warning
+
+Naming/AccessorMethodName:
+  Enabled: false
+
+Documentation/DocumentationAdmonition:
+  Enabled: false
```

Comment:
What things are these disabling?

Reply:
It fails with Ameba 1.6. This change is no longer needed after [this commit](https://github.com/crystal-loot/selenium.cr/pull/53/commits/5dabfd25fccc4df50b29af2f417949bf5d4649d9). I’ve reverted it.